src/database-sqlite.ts

```diff
@@ -0,0 +1,33 @@
+import { path, sqlite } from './deps.ts'
+
+const { DB } = sqlite
+
+let db = null
+let databaseFilename
+let writeStatement
+
+export function checkKey (key: string): void {
+  if (/[/\\]/.test(key)) {
+    throw new Error(`bad key: ${key}`)
+  }
+}
```

Comment:
Should this also be updated with the latest `checkKey` code from the other PR?

Reply:
Yes, good catch